CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+Changelog

```

Comment:
Can we say "release notes" instead? (also for the file name)

Reply:
I have picked it from few Jenkins suggested example plugins like https://github.com/jenkinsci/envinject-api-plugin . CHANGELOG also is kind of a simple one full name. if should this be renamed then i have two options. which one is preferred ?
 RELEASENOTES 
 RELEASE_NOTES

Reply:
Thanks for the info. CHANGELOG is fine. We can discuss the name in our next group meeting.